frontend/src/pages/modelServing/screens/projects/utils.ts

```diff
@@ -667,3 +667,34 @@ export const getCreateInferenceServiceLabels = (
   }
   return undefined;
 };
+
+export const getPVC = async (
+  namespace: string,
+  editInfo: {
+    servingRuntimeEditInfo?: {
+      servingRuntime?: ServingRuntimeKind;
+    };
+  },
```

Comment:
Curious why you don't just take the runtime? Why the extra structure?

Reply:
Updated